vta/python/vta/testing/simulator.py

```diff
@@ -74,5 +74,14 @@ def tsim_init(hw_lib):
     m = tvm.module.load(lib, "vta-tsim")
     f(m)
 
+def tsim_cycles():
+    """Clear profiler statistics
```

Comment:
Does this actually clear profiler statistics? Or return the profiler stats?

Reply:
I forgot to update the function doc. Fixed.